src/domUpdates.js

```diff
@@ -21,28 +23,31 @@ let domUpdates = {
 
   displayTotalTravelerSpendings(total) {
     let totalSpent = document.querySelector(".total-spent");
+    totalSpent.innerText = ""
     totalSpent.innerText =
-      `You have spent $${total} this year.`
+      `You have spent $${total} in 2020.`
   },
 
   displayCurrentTravelerTrip(traveler, destinations) {
     let current = document.querySelector(".current-trip");
+    current.innerHTML = "";
     if (traveler.currentTrips[0] === undefined) {
       current.innerHTML = `<p>You shouldn\'t be on a trip right now. Maybe you should plan your next one if you don\'t have any upcoming trips either!!</p>`
     } else {
       let foundDest = findDestinationBasedOnTrip(traveler.currentTrips[0], destinations);
       let splitDestName = foundDest.destination.split(", ");
       let dateSplit = traveler.currentTrips[0].date.split("/");
       current.innerHTML = `<div class="current card">
-        <p>this is a place holder for the image</p>
+        <img class="card-image" src="${destination.image}" alt="${destination.alt}">
         <h4 class="trip-destination">${splitDestName[0]},<br> ${splitDestName[1]}</h4>
-        <p class="details">Trip start date: ${dateSplit[1]}/${dateSplit[2]}/${dateSplit[0]}<br> Number of travelers: ${traveler.currentTrips[0].travelers}</p>
+        <p class="details">Trip start date: ${dateSplit[1]}/${dateSplit[2]}/${dateSplit[0]}<br> Duration of trip: ${trip.duration} <br> Number of travelers: ${traveler.currentTrips[0].travelers}</p>
       </div>`
     }
   },
 
   displayUpcomingTrips(traveler, destinations) {
     let upcoming = document.querySelector(".upcoming-trips-container");
+    upcoming.innerHTML = "";
     if (traveler.upcomingTrips[0] === undefined) {
       upcoming.innerHTML = `<p>You don\'t have any upcoming trips, click PLAN A TRIP to plan your next trip!!</p>`
```

Comment:
I like how well you thought the display through.  The messages are very helpful with navigation and knowing what to expect to see.